src/Ocelot/Request/Mapper/StreamHttpContent.cs

```diff
@@ -56,7 +56,7 @@ private static async Task CopyAsync(Stream input, Stream output, long announcedC
                 if (zeroByteReadTask.IsCompletedSuccessfully)
                 {
                     // Consume the ValueTask's result in case it is backed by an IValueTaskSource
-                    _ = zeroByteReadTask.Result;
+                    _ = zeroByteReadTask.GetAwaiter().GetResult();
```

Comment:
It appears that reading `Result` once is acceptable when the value task is complete, as indicated by the [ValueTask{TResult}.Result Property | Remarks](https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask-1.result?view=net-8.0#remarks). Therefore, this change may be unnecessary due to the assurance provided by `.IsCompletedSuccessfully`.
@ggnaegi, would you agree?

Reply:
This change is not required. See details above.

Reply:
Fixed in https://github.com/ThreeMammals/Ocelot/pull/2156/commits/8b107922ce3aa4e6941ea21c0ff7bc62912e6507